docs/conf/portland/2025/virtual.md

```diff
@@ -0,0 +1,39 @@
+---
+template: {{year}}/generic.html
+---
+
+# Virtual Attendance
+
+We're excited to offer a virtual attendance option for Write the Docs Portland 2025.
+The virtual component will run on **May 5-6** alongside our in-person conference.
+
+## Platform
+
+We'll be hosting the virtual conference on [Venueless](https://venueless.org), 
+a platform that combines video streaming with chat and networking features.
+
+## What's Included
+
+Virtual attendance includes:
+
+* Live streamed talks from the main stage
+* Real-time chat with all attendees
```

Comment:
I don't think in-person attendees are going to be online?
```suggestion
* Real-time chat with other virtual attendees
```

Reply:
That's actually an interesting point. If we keep the chat in something like slack its more likely that folks will be there from the in person conference.